packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/StylesFactory.java

```diff
@@ -152,7 +152,7 @@ public INodeStyle createNodeStyle(NodeStyleDescription nodeStyle, Optional<Strin
                                          .filter(FixedColor.class::isInstance)
                                          .map(FixedColor.class::cast)
                                          .map(FixedColor::getValue)
-                                         .orElse(DEFAULT_COLOR))
+                                         .orElse(""))
```

Comment:
Can't we apply this same behavior on the color properties? Actually, no value for color means black background, I expected a transparent one.
If we remove the the _DEFAULT_COLOR_ it doesn't work on legacy diagram because the opacity is set to 1 but on react flow diagram it working correctly I have no background.

Reply:
Yes we could but it should be part of a separate ticket/PR